src/openrct2-ui/windows/TitleMenu.cpp

```diff
@@ -204,7 +204,7 @@ static void window_title_menu_dropdown(rct_window* w, rct_widgetindex widgetInde
                 Editor::LoadTrackManager();
                 break;
             case 4:
-                context_open_custom_user_content_folder();
+                context_open_custom_content_folder();
```

Comment:
Hm, I see you followed a remark that I posted, but I removed that within a minute, since I wasn't sure. Sorry about that.

I'll look into it myself, since I'll have to add Linux support anyway.

Reply:
Oh, I got an email about it, so I just thought I was blind haha.

Reply:
Can just revert the commit if necessary :)

Reply:
Nah, I'll do that. It's not worth refiring the CI just to remove the commit - I'll do it when I do the definitive rename and add Linux support.